faq/templates/faq/faqlist.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'web/base.html' %}
+{% load i18n %}
+{% load static %}
+
+
+{% block head %}
+    <link rel="stylesheet" type="text/css" href="{% static "faq/css/faqlist.css" %}"/>
```

Comment:
The `type` attribute is redundant, as the browser can infer it from the `rel` attribute (see [this page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-type)).
```suggestion
    <link rel="stylesheet" href="{% static "faq/css/faqlist.css" %}"/>
```